x-pack/plugins/security_solution/server/lib/detection_engine/routes/signals/open_close_signals_route.ts

```diff
@@ -170,9 +180,16 @@ const updateSignalsStatusByQuery = async (
     ignore_unavailable: true,
   });
 
-const getUpdateSignalStatusScript = (status: SetSignalsStatusSchemaDecoded['status']) => ({
-  source: `if (ctx._source['${ALERT_WORKFLOW_STATUS}'] != null) {
-      ctx._source['${ALERT_WORKFLOW_STATUS}'] = '${status}'
+const getUpdateSignalStatusScript = (
```

Comment:
This only returns user profiles that are `enabled` right? Is a profile enabled automatically on login? Would there be a case of someone being logged onto the UI with a disabled profile and so this status is not logged?